### PR TITLE
swagger for delete workspaces seems to have been forgotten

### PR DESCRIPTION
**Pull request by helgridly**
Easy peasy.

### CODE REVIEW

**Comment by helgridly** [comment]
I just added a fix for a match error in delete workspace too.

**Comment by dvoet** [comment]
+1

File: src/main/resources/swagger/listings/workspaces [hunk 1]
```diff
@@ -198,12 +198,44 @@
...
 				"code": 500,
 				"message": "Rawls Internal Error"
 			}]
-		}]
+		}, {
+            "method": "DELETE",
+            "summary": "Delete workspace",
+            "notes": "",
+            "type": "Workspace",
+            "nickname": "list",
```
Comment (tedsharpe) [inline src/main/resources/swagger/listings/workspaces]: Silly quibbles, but this method doesn't actually return a Workspace, and the nickname is wrong.

**Comment by helgridly** [comment]
- [ ] testing
- [ ] checklists
- [x] they work

**Comment by MatthewBemis** [comment]
+1 from me too

**Pull request merged**